test/src/main/java/org/openjdk/skara/test/TestIssue.java

```diff
@@ -91,7 +91,7 @@ public void setTitle(String title) {
 
     @Override
     public String body() {
-        return store.body();
+        return body;
```

Comment:
Make `TestIssue#body()` consistent with `GitHubPullRequest#body()` and `GitLabMergeRequest#body()`.

It was returning the latest body from the 'remote' instead of the cache. So it is different from `GitHubPullRequest` and `GitLabMergeRequest`

Reply:
In effect, it sounds like `TestIssue` did not properly mimick the real classes. So fixing this, and fixing the tests so they pass, is probably a good idea. Maybe you should do that separately, yes, and integrate that first. Then you can easily add a proper test to this PR.

Reply:
> In effect, it sounds like `TestIssue` did not properly mimick the real classes. So fixing this, and fixing the tests so they pass, is probably a good idea. Maybe you should do that separately, yes, and integrate that first. Then you can easily add a proper test to this PR.

Got it! Thx!